var/spack/repos/builtin/packages/py-xtb/package.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyXtb(PythonPackage):
+    """Python API for the extended tight binding program package"""
+
+    homepage = "https://xtb-python.readthedocs.org"
+    url = "https://github.com/grimme-lab/xtb-python/releases/download/v22.1/xtb-22.1.tar.gz"
+
+    maintainers = [
+        *PythonPackage.maintainers,
+        "awvwgk",
+    ]
+
+    version("22.1", sha256="7a59e7b783fc6e8b7328f55211de681e535a83991b07c4bab73494063f5e9018")
+
+    depends_on("pkgconfig", type="build")
+    depends_on("python@3.7:", type=("build", "run"))
+    depends_on("py-cffi", type=("build", "run"))
+    depends_on("py-meson-python@0.11.0:", type="build")
```

Comment:
Where does 0.11+ come from?

Reply:
I have followed the development of mesen-python  and it has been robustly usable for most of my projects since the 0.11 release. I can try to figure out if older versions will build successfully as well.

Reply:
I would remove the constraint entirely and stick to what the developers require (no version constraint) unless you find evidence otherwise.